src/databricks/labs/remorph/reconcile/query_builder_refactored/recon_config.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from pyspark.sql import DataFrame
+
```

Comment:
why is this file package called, `query_builder_refactored`?

Reply:
we have the old code base and new code base based on sqlglot expression.so once we move everything we will remove the older one and rename the query_builder_refactored to query_builder.Because we have dependencies that can't be refactored within the same package.